packages/mc-scripts/src/config/create-webpack-config-for-production.js

```diff
@@ -416,6 +416,9 @@ module.exports = function createWebpackConfigForProduction(options = {}) {
             {
               loader: require.resolve('babel-loader'),
               options: {
+                ...(Number.isInteger(mergedOptions.toggleFlags.parallelism)
```

Comment:
We check for a number cause for Terser we accept a Boolean which thread-loader doesn't accept.